specification/automation/resource-manager/Microsoft.Automation/preview/2023-05-15-preview/examples/job/getJobOutput.json

```diff
@@ -0,0 +1,17 @@
+{
+  "parameters": {
+    "subscriptionId": "51766542-3ed7-4a72-a187-0c8ab644ddab",
+    "resourceGroupName": "mygroup",
+    "automationAccountName": "ContoseAutomationAccount",
+    "jobName": "foo",
+    "api-version": "2023-05-15-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Content-Type": "text/plain"
+      },
+      "body": "Wednesday, February 7, 2018 3:47:17 PM"
```

Comment:
>body [](http://example.com/codeflow?start=7&length=4)

Having a string response is not easy to extend in the future without breaking clients. I'd suggest you change it to a JSON response unless there's a strong reason not to